### PR TITLE
WRN-1347: Migrate ContextualPopupDecorator unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for ContextualPopupDecorator component.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1347

### CODE REVIEW

File: ContextualPopupDecorator/tests/ContextualPopupDecorator-specs.js [hunk 1]
```diff
@@ -1,77 +1,95 @@
...
-				return null;
-			}
-		}
-
-		const subject = mount(
-			<ContextualButton
-				popupComponent={Button}
-			>
-				Button
-			</ContextualButton>
```
Comment (adrian-cocoara-lgp) [inline ContextualPopupDecorator/tests/ContextualPopupDecorator-specs.js:24]: Note to next reviewer: This is a kind of test which refers to implementation detail of a component. A user does not see this action happening.

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by MikyungKim — APPROVED** [review]
Nice work!

**Pull request merged**